arches/app/media/css/pages/_search.scss

```
@@ -0,0 +1,193 @@
+article.main-search-container {
+    @include break-at(desktop-lg) {
+        height: 100%;
+        overflow-x: hidden;
+    }
+    @include break-at(desktop-max) {
+        flex-direction: column;
+    }
+    .search-results-panel {
+        @include break-at(desktop-max) {
+            max-width: calc(100vw - 50px);
+        }
+        .search-tools-container {
+            @include break-at(desktop-max) {
+                width: 100%;
+            }
+        }
+        .search-control-container {
+            @include break-at(desktop-max) {
+                overflow-y: auto;
+            }
+            @include break-at(desktop-lg) {
+                flex: 1;
+            }
+            .search-results-container {
+                @include break-at(desktop-max) {
+                    width: 100%;
+                }
+                @include break-at(desktop-lg) {
+                    padding: 10px 10px 20px 10px;
+                    overflow-y: auto;
+                }
+                .search-listing {
+                    @include break-at(desktop-max) {
+                        width: 100%;
+                    }
+                }
+            }
+        }
+    }
+    .search-map-container {
+        flex-direction: column;
+        .search-toolbar {
+            @include break-at(desktop-max) {
+                border-top: solid 1px #bbd1ea;
+                height: 52px;
+            }
+            @include break-at(desktop-max) {
+                flex-direction: column;
+            }
+            .search-type-btn-panel {
+                @include break-at(desktop-max) {
+                    margin-left: 5px;
+                }
+                @include break-at(tablet-max) {
+                    display: flex;
+                }
```

At screen-sizes > 915px, could the search-toolbar form a single row? 

```suggestion
        .search-toolbar {
            @include break-at(desktop-max) {
                border-top: solid 1px #bbd1ea;
                height: auto;
            }
            @include break-at(desktop-lg) {
                flex-direction: column;
                height: 52px;
            }
            .search-type-btn-panel {
                @include break-at(desktop-max) {
                    margin-left: 5px;
                    height: auto;
                }
```

---

Fixed in commit 63ab038

---

Suggestion used and fixed in commit a6cdaa2